docs/ai/quickstarts/build-mcp-server-publish-nuget.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Quickstart - Create a minimal MCP Server using .NET and publish to NuGet
+description: Learn to create and connect to a minimal MCP server using .NET and publish it to NuGet.
+ms.date: 06/30/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: jondouglas
+ms.author: jondouglas
+---
+
+# Create, connect and publish a minimal MCP server using .NET to NuGet
```

Comment:
Should we have a warning up here about this being a preview experience? Notably a preview C# MCP SDK (API shape may change) and needing preview .NET 10 SDK for now.

Reply:
i dont think we need to given how far along the release is already